contrib/platform/src/com/sun/jna/platform/win32/Crypt32.java

```diff
@@ -109,4 +110,27 @@ public boolean CryptUnprotectData(DATA_BLOB pDataIn, PointerByReference szDataDe
 			CRYPTPROTECT_PROMPTSTRUCT pPromptStruct,
 			int dwFlags,
 			DATA_BLOB pDataOut);
+
+	/**
+	 * The CertAddEncodedCertificateToSystemStore function opens the specified
+	 * system store and adds the encoded certificate to it.
+	 * 
+	 * @param szCertStoreName
+	 *            A null-terminated string that contains the name of the system
+	 *            store for the encoded certificate.
+	 * @param pbCertEncoded
+	 *            A pointer to a buffer that contains the encoded certificate to
+	 *            add.
+	 * @param cbCertEncoded
+	 *            The size, in bytes, of the pbCertEncoded buffer.
+	 * @return If the function succeeds, the return value is TRUE.<br>
+	 *         If the function fails, the return value is FALSE.<br>
+	 *         CertAddEncodedCertificateToSystemStore depends on the functions
+	 *         listed in the following remarks for error handling. <br>
+	 *         Refer to those function topics for their respective error
+	 *         handling behaviors.<br>
+	 *         For extended error information, call GetLastError.
+	 * @see MSDN {@link http://msdn.microsoft.com/en-us/library/bb736347(v=vs.85).aspx }
+	 */
+	boolean CertAddEncodedCertificateToSystemStore(String szCertStoreName, Pointer pbCertEncoded, DWORD cbCertEncoded);
```

Comment:
You can usually safely use _int_ instead of _DWORD_ here (as well as for return values). I agree that DWORD and _int_ are not entirely equivalent, but for most purposes the sign is not important - e.g., error codes or sizes (as in this case).